static/js/components/Toast.js

```diff
@@ -30,7 +30,7 @@ export default class Toast extends React.Component {
   render() {
     const { children, open } = this.props;
 
-    return <div className={`toast ${open ? 'open' : ''}`}>
+    return <div role="alert" className={`toast ${open ? 'open' : ''}`}>
```

Comment:
Oh, good point! However, wouldn't it be even better not to render the `<Toast>` component at all if it's not active?